security/medusa/l2/kobject_process.c

```diff
@@ -29,7 +29,7 @@ medusa_answer_t process_kobj2kern(struct process_kobject *tk, struct task_struct
 
 		new_user = alloc_uid(tk->uid);
 		if (!new_user)
-			return MED_DENY;
+			return MED_ERR;
```

Comment:
as process_kobj2kern() returns int, should return i.e. -1
it's used only once  in process_update(), and in this function is made conversion from process_kobj2kern() return code to medusa_answer_t